### DISCUSSION TITLE
Replace characters in PDF file

### DISCUSSION THREAD

**Discussion by toniojst** [nextgenhealthcare/connect#5695] [Q&A]
Hi, i’m wondering how and if i can replace characters of alredy created PDF file with help of some java library like this one (https://stackoverflow.com/questions/55451551/unable-to-save-arabic-words-in-a-pdf-pdfbox-java)?
Will this work or not?

KR. Toni

**Comment by kirbykn2** [comment]
Did you try it?  What was the result?

**Reply by toniojst** [reply]
Not yet. I must do this but i didnt try jet so post question here if someone have experince with this.

**Reply by toniojst** [reply]
Or i find another example which i will also try if will no someone reply with better and worked solution. https://kb.itextpdf.com/home/it5kb/examples/replacing-pdf-objects
